services/web-app/components/glossary/glossary-list/glossary-list.module.scss

```diff
@@ -36,7 +32,16 @@
   font-style: italic;
 }
 
-.page-h4 {
-  margin-top: spacing.$spacing-04;
-  @include type.type-style('heading-02');
+.h2-container {
+  &.h2-container {
+    margin-top: 0;
+  }
+
+  + * {
+    margin-top: 0;
+  }
+}
+
+.h4-container.h4-container {
+  margin-top: spacing.$spacing-06;
```

Comment:
We should try to be consistent with how we write these, either using nested selectors for both or single-line selectors for both. My preference would be nested selectors.

Reply:
Agree on consistency, however I prefer single line, I think its easier to read.

Reply:
made all nested, since turning the nested once into single lines would result in slight duplication